docs/docs/user_guide/secure_install.md

```diff
@@ -31,7 +31,7 @@ chmod +x ./install_aac_verified_index.bash
 ```
 
 ## Air-Gapped Installation
-For air-gapped installations without access to PyPI, or which may not want to use the indexed artifacts, we provide a PyPI-less (no index) installation method. You can execute `pip install --require-hashes -r requirements.txt --no-index --find-links ./` or the `install_aac_air_gap` script. This mode will verify the hashes of the AaC and its runtime dependencies, packaged as wheels, and then install them -- the runtime artifacts are not pulled from PyPI.
+For air-gapped installations without access to PyPI, or which may not want to use the indexed artifacts, we provide a PyPI-less (no index) installation method. You can execute `pip install --require-hashes -r requirements.txt --no-index --find-links ./` or the `install_aac_air_gap` script. This mode will verify the hashes of the AaC and its runtime dependencies, packaged as wheels in the secure installation archive, and then it will install them. **The runtime artifacts are not pulled from PyPI.**
```

Comment:
```suggestion
For air-gapped installations without access to PyPI, or which may not want to use the indexed artifacts, we provide a PyPI-less (no index) installation method. You can execute `pip install --require-hashes -r requirements.txt --no-index --find-links ./` or the `install_aac_air_gap` script. This mode will verify the hashes of the AaC package and its runtime dependencies, packaged as wheels in the secure installation archive, and then it will install them. **The runtime artifacts are not pulled from PyPI.**
```